egs/hkust/s5/local/hkust_prepare_dict.sh

```diff
@@ -31,7 +31,7 @@ cat $dict_dir/words.txt | grep -v '[a-zA-Z]' > $dict_dir/lexicon-ch/words-ch.txt
 ##### produce pronunciations for english 
 if [ ! -f $dict_dir/cmudict/cmudict.0.7a ]; then
   echo "--- Downloading CMU dictionary ..."
-  svn co https://svn.code.sf.net/p/cmusphinx/code/trunk/cmudict \
+  svn co -r 13068 https://svn.code.sf.net/p/cmusphinx/code/trunk/cmudict \
```

Comment:
I'll updating the results today.


在 2016/6/28 21:25, Vijayaditya Peddinti 写道:
>
> In egs/hkust/s5/local/hkust_prepare_dict.sh 
> <https://github.com/kaldi-asr/kaldi/pull/871#discussion_r68755258>:
>
> > @@ -31,7 +31,7 @@ cat $dict_dir/words.txt | grep -v '[a-zA-Z]' > $dict_dir/lexicon-ch/words-ch.txt
> >  ##### produce pronunciations for english
> >  if [ ! -f $dict_dir/cmudict/cmudict.0.7a ]; then
> >    echo "--- Downloading CMU dictionary ..."
> > -  svn co https://svn.code.sf.net/p/cmusphinx/code/trunk/cmudict \
> > +  svn co -r 13068 https://svn.code.sf.net/p/cmusphinx/code/trunk/cmudict \
>
> @naxingyu <https://github.com/naxingyu> are the latest results in 
> hkust based on this version of the dictionary ?
> fisher_swbd results are most probably based on this dictionary as 
> @naxingyu <https://github.com/naxingyu> ran them recently. ASpIRE 
> results are based on an older version of the dictionary and I am 
> pretty sure it is same case with fisher_english results. However I am 
> merging this pull request as we will later update these older results.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/871/files/e661947b3ea5b6ad41e95439b3cec31389e99d49#r68755258>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe/ADKpxEzIv3P4D3RfZ5acBEuITxljgS5dks5qQSCxgaJpZM4I_riX>.
>